### PR TITLE
Update 1-4 blog post

### PR DESCRIPTION
**Pull request by jaredpalmer**

### CODE REVIEW

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+
+This improvement in hash specificity by framework is a significant step toward optimal, safe, and correct caching.
+
+## `eslint-plugin-turbo`
+
+We've also created a new ESLint plugin `eslint-plugin-turbo` to provide further in-editor assistance and conformance to help ensure your Turborepo cache can be correctly shared across every environment. While our new hashing algorithm should cover most situations with most frameworks, our new plugin will provide feedback for teams using other build time inlined environment variables that are not framework-prefixed which impact build outputs (i.e. caching) and teams using their own in-house framework which we don't infer automatically.
+
+To get started, simply extend from `eslint-config-turbo` in your root `[eslintrc](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-file-formats)` file:
```
Comment (leerob) [inline docs/pages/blog/turbo-1-4-0.mdx:70]:
```suggestion
To get started, simply extend from `eslint-config-turbo` in your root [`eslintrc`](https://eslint.org/docs/latest/user-guide/configuring/configuration-files#configuration-file-formats) file:
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+
+```jsx
+{
+	// Automatically flag env vars missing from turbo.json
+  "extends": ["next/core-web-vitals", "turbo"],
```
Comment (leerob) [inline docs/pages/blog/turbo-1-4-0.mdx:75]:
```suggestion
  // Automatically flag env vars missing from turbo.json
  "extends": ["next/core-web-vitals", "turbo"],
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+
+```jsx
+{
+
+  "extends": ["next/core-web-vitals"],
+	"plugins": ["turbo"]
+  "rules": {
+  	// Automatically flag env vars missing from turbo.json
+    "turbo/no-undeclared-env-vars": "error"
+  }
```
Comment (leerob) [inline docs/pages/blog/turbo-1-4-0.mdx:89]:
```suggestion
  "extends": ["next/core-web-vitals"],
  "plugins": ["turbo"]
  "rules": {
    // Automatically flag env vars missing from turbo.json
    "turbo/no-undeclared-env-vars": "error"
  }
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 4]
```diff
@@ -0,0 +1,126 @@
...
+# Turborepo 1.4
+
+import { Authors } from '../../components/Authors'
+import Callout from '../../components/Callout'
+
+<div className="text-sm mt-2 text-center text-gray-500 dark:text-gray-400">Tuesday, August 9th, 2022</div>
+
+<Authors authors={[ 'gsoltis', 'nathanhammond', 'tknickman', 'anthonyshewdev' ]} />
+
+Turborepo 1.4 improves usage with frameworks and adds new examples:
```
Comment (mehulkar) [inline docs/pages/blog/turbo-1-4-0.mdx:18]:
```suggestion
Turborepo 1.4 brings:
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 5]
```diff
@@ -0,0 +1,126 @@
...
+
+- [**Automatic environment variable inclusion:**](#automatic-environment-variable-inclusion): We'll automatically infer popular framework environment variables for you. No need to declare them yourself in `turbo.json`.
+- [**`eslint-plugin-turbo`**](#eslint-plugin-turbo): Enhanced feedback with a new ESLint plugin
+- [**New framework and library examples:**](#new-framework-and-library-examples) New starters and examples requested by the community.
+
+Update today by running `npm install turbo@latest`.
+
+## Automatic environment variable inclusion
+
+To help ensure correct caching across environments, Turborepo will now automatically infer and include public environment variables when calculating cache keys for apps built with Astro, Create React App, Gatsby, Next.js, Nuxt, SvelteKit, Vite, Vue, and more. You can safely remove these framework-specific public environment variables from `turbo.json` if you have manually declared them.
```
Comment (mehulkar) [inline docs/pages/blog/turbo-1-4-0.mdx:28]:
```suggestion
To help ensure correct caching across environments, Turborepo will now automatically infer and include public environment variables when calculating cache keys for apps built with Astro, Create React App, Gatsby, Next.js, Nuxt, SvelteKit, Vite, Vue, and more. You can safely remove framework-specific public environment variables from `turbo.json` if you manually declared them.
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 6]
```diff
@@ -0,0 +1,126 @@
...
+```jsonc filename="turbo.json"
+{
+  "pipeline": {
+    "build": {
+      "dependsOn": [
+				"^build",
+			]
+    }
+}
+```
```
Comment (mehulkar) [inline docs/pages/blog/turbo-1-4-0.mdx:60]: source looks ok, but indent is weird. not a blocker
Screen Shot 2022-08-08 at 4 51 25 PM

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 7]
```diff
@@ -0,0 +1,126 @@
...
+
+As of 1.4, when you run `turbo run build`, Turborepo will not consider any build time environment variables relevant when building the TypeScript packages. However, when building the Next.js app, Turborepo will infer that environment variables starting with `NEXT_PUBLIC_` could alter the output of the `.next` folder and should thus be included when calculating the hash. Similarly, when calculating the hash of the Create React App's `build` script, all build time environment variables starting with `REACT_APP_PUBLIC_` will be included.
```
Comment (mehulkar) [inline docs/pages/blog/turbo-1-4-0.mdx:62]:
```suggestion
As of 1.4, when you run `turbo run build`, Turborepo will not consider any build time environment variables relevant when building the TypeScript package. However, when building the Next.js app, Turborepo will infer that environment variables starting with `NEXT_PUBLIC_` could alter the output of the `.next` folder and should thus be included when calculating the hash. Similarly, when calculating the hash of the Create React App's `build` script, all build time environment variables starting with `REACT_APP_PUBLIC_` will be included.
```

File: docs/pages/blog/turbo-1-4-0.mdx [hunk 8]
```diff
@@ -0,0 +1,126 @@
...
+
+This improvement in hash specificity by framework is a significant step toward optimal, safe, and correct caching.
+
+## `eslint-plugin-turbo`
```
Comment (mehulkar) [inline docs/pages/blog/turbo-1-4-0.mdx:66]: if we recommend `eslint-config-turbo`, should the heading be config instead of plugin?

**Review by mehulkar — COMMENTED** [review]
some suggestions, but mostly 🚢 !

**Review by mehulkar — APPROVED** [review]

**Pull request merged**